lib/components/Button/index.jsx

```diff
@@ -3,18 +3,20 @@ import PropTypes from 'prop-types';
 
 const Button = ({
   label,
+  id,
   classes,
   type,
   onClick,
 }) => (
-  <button type={type} className={`btn ${classes}`} onClick={onClick}>{label}</button>
+  <button id={id} type={type} className={`btn ${classes}`} onClick={onClick}>{label}</button>
 );
 
 // eslint-disable-next-line
 Button.propTypes = {
   label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
   classes: PropTypes.string,
   type: PropTypes.string,
+  id: PropTypes.string.isRequired,
```

Comment:
Aren't we going to get loads of failures/warnings when whatever uses this now does not provide the id?

Reply:
Warnings, not failures - and yes, but as we're also saying that we want all user controls to have identifiers so that UI testing is easier, this doesn't feel like a bad thing to be honest.